src/config/mod.rs

```diff
@@ -84,14 +85,21 @@ impl Configs {
 
         let map: HashMap<String, serde_json::value::Value> = s.try_into()?;
 
-        let config = &map[name.as_ref()];
+        let config = match map.get(name.as_ref()) {
+            Some(c) => c,
+            None => bail!(
+                "Cannot find config \"{}\" (available keys: {})",
```

Comment:
Maybe do not use `keys` here and rather refer to it as `config` or similar. At least we should use consistent terms I think. I know it's the key associated with a config, but it could be confusing.

Reply:
Fixed!